src/Game/Game.js

```diff
@@ -56,13 +55,23 @@ function Board({ xIsNext, squares, onPlay }) {
 export default function Game() {
   const [history, setHistory] = useState([Array(9).fill(null)]);
   const [currentMove, setCurrentMove] = useState(0);
+  const [playAgainstComputer, setPlayAgainstComputer] = useState(false);
   const xIsNext = currentMove % 2 === 0;
   const currentSquares = history[currentMove];
 
   function handlePlay(nextSquares) {
     const nextHistory = [...history.slice(0, currentMove + 1), nextSquares];
     setHistory(nextHistory);
     setCurrentMove(nextHistory.length - 1);
+
+    if (playAgainstComputer && !calculateWinner(nextSquares)) {
+      const computerMove = findBestMove(nextSquares);
+      const newSquares = nextSquares.slice();
+      newSquares[computerMove] = "X";
+
+      setHistory([...nextHistory, newSquares]);
+      setCurrentMove(nextHistory.length);
+    }
```

Comment:
This code is checking if the computer is playing and then taking the computer turn by calling findBestMove function